src/main/java/com/aws/greengrass/deployment/ShadowDeploymentListener.java

```diff
@@ -56,29 +75,16 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.concurrent.locks.Lock;
-import javax.inject.Inject;
 
-import static com.aws.greengrass.deployment.DeploymentService.DEPLOYMENT_DETAILED_STATUS_KEY;
-import static com.aws.greengrass.deployment.DeploymentService.DEPLOYMENT_ERROR_STACK_KEY;
-import static com.aws.greengrass.deployment.DeploymentService.DEPLOYMENT_ERROR_TYPES_KEY;
-import static com.aws.greengrass.deployment.DeploymentService.DEPLOYMENT_FAILURE_CAUSE_KEY;
-import static com.aws.greengrass.deployment.DeploymentStatusKeeper.CONFIGURATION_ARN_KEY_NAME;
-import static com.aws.greengrass.deployment.DeploymentStatusKeeper.DEPLOYMENT_STATUS_DETAILS_KEY_NAME;
-import static com.aws.greengrass.deployment.DeploymentStatusKeeper.DEPLOYMENT_STATUS_KEY_NAME;
-import static com.aws.greengrass.deployment.model.Deployment.DeploymentType;
-import static com.aws.greengrass.status.model.DeploymentInformation.ARN_FOR_STATUS_KEY;
-import static com.aws.greengrass.status.model.DeploymentInformation.STATUS_DETAILS_KEY;
-import static com.aws.greengrass.status.model.DeploymentInformation.STATUS_KEY;
-import static com.aws.greengrass.status.model.StatusDetails.DETAILED_STATUS_KEY;
-import static com.aws.greengrass.status.model.StatusDetails.ERROR_STACK_KEY;
-import static com.aws.greengrass.status.model.StatusDetails.ERROR_TYPES_KEY;
-import static com.aws.greengrass.status.model.StatusDetails.FAILURE_CAUSE_KEY;
+import javax.inject.Inject;
 
 @NoArgsConstructor
 public class ShadowDeploymentListener implements InjectionActions {
 
-    private static final long TIMEOUT_FOR_SUBSCRIBING_TO_TOPICS_SECONDS = Duration.ofMinutes(1).getSeconds();
-    private static final long TIMEOUT_FOR_PUBLISHING_TO_TOPICS_SECONDS = Duration.ofMinutes(1).getSeconds();
+    private static final long TIMEOUT_FOR_SUBSCRIBING_TO_TOPICS_SECONDS =
+            Duration.ofMinutes(1).getSeconds();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Please verify the usage of duration getter methods `getNano()` and `getSeconds()`. If you need the total duration, use conversion methods like `toNanos()` or `toSeconds()` (Java 9+) instead. For Java 8, use `getSeconds` to get total seconds, as `toSeconds` is not available. These methods ensure you get the complete duration value, whereas getters may not always provide the full duration.